packages/trader/src/Modules/Reports/Containers/profit-table.jsx

```diff
@@ -46,38 +46,33 @@ class ProfitTable extends React.Component {
             );
         }
 
+        // Remove 'Type' title in mobile mode
+        const column_clone = { ...this.columns_map };
```

Comment:
Why did we clone this?

Reply:
To take a new object with `type` property removed and make sure we do not change/corrupt the initial data that was set to this object.

Reply:
Can we instead set `title` in `data-table-constants.js ` based on `isMobile()` check?

Reply:
Yea sure. A better and cleaner solution.